### DISCUSSION TITLE
App tracking transparency requirements

### DISCUSSION THREAD

**Discussion by qburst-subins** [invertase/react-native-firebase#5217] [Q&A] [answered]
We are using Firebase services like Crashlytics, Analytics and Cloud messaging in our application. For analytics we have only sending user id(created from Backend) as a property.
And we are not using these data for any advertisements. So do we need to integrate ATT in our app?

**Comment by mikehardy** [comment]
I would consult the upstream firebase-ios-sdk documentation / issue repo and/or ask on stackoverflow (where the actual firebase group posts answers if you tag things appropriately). Both of those will be more definitive than any answers we provide

**Comment by edreyyo** [comment]
Did you make any progress on this, @subinsuresh? Our app is getting rejected by the iOS review team because of ATT but we have no idea what package is triggering the fetching of the IDFA.

**Comment by mikehardy** [comment, marked as answer]
Podfile.lock is how you inspect your dependencies, and you'll see that Analytics pulls in Firebase/Analytics and GoogleAppMeasurement - those both use IDFA up to firebase-ios-sdk 7.10.0. I imagine you will really like react-native-firebase 11.4.1 (using firebase-ios-sdk 7.11.0) - https://github.com/invertase/react-native-firebase/blob/master/CHANGELOG.md#1140-2021-04-29

> analytics, ATT: allow use of AnalyticsWithoutAdIdSupport pod (da6b811)

https://rnfirebase.io/analytics/usage#disable-ad-id-usage-on-ios

**Reply by edreyyo** [reply]
Thanks @mikehardy. Upgrading to the latest versions of `react-native-firebase` along with using the `$RNFirebaseAnalyticsWithoutAdIdSupport = true` flag did the trick.
